antarest/study/business/model/renewable_cluster_model.py

```diff
@@ -0,0 +1,67 @@
+# Copyright (c) 2025, RTE (https://www.rte-france.com)
+#
+# See AUTHORS.txt
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+# SPDX-License-Identifier: MPL-2.0
+#
+# This file is part of the Antares project.
+
+from antares.study.version import StudyVersion
+
+from antarest.core.utils.string import to_kebab_case
+from antarest.study.business.all_optional_meta import all_optional_model, camel_case_model
+from antarest.study.storage.rawstudy.model.filesystem.config.renewable import (
+    RenewableConfig,
+    RenewableProperties,
+    RenewablePropertiesType,
+    create_renewable_properties,
+)
+
+
+@all_optional_model
+@camel_case_model
+class RenewableClusterUpdateDTO(RenewableProperties):
```

Comment:
Better to remove the `DTO` suffix, this is a business layer class, it should not have any technical suffix.